apphandler.cpp

```diff
@@ -648,7 +637,12 @@ ipmi::RspType<uint8_t,  // Device ID
                 devId.addnDevSupport = data.value("addn_dev_support", 0);
                 devId.manufId = data.value("manuf_id", 0);
                 devId.prodId = data.value("prod_id", 0);
-                devId.aux = data.value("aux", 0);
+                // For VERSION_ID = op940.20-2-0-g414986cad the first byte in
+                // the auxiliary version is Service Pack version 20, the second
+                // byte is IBM's revision 2, the third byte is IBM's extended
+                // version 0 and the fourth byte 0x4F indicates it is an
+                // OpenPower build.
```

Comment:
0x4F = "O", I assume we're using the letter "O" as an arbitrary value to denote an "OpenPower build"

Reply:
Yes @anoo1 .

0x46 for 'F' representing FW and 0x4F for 'O' representing OP